### PR TITLE
clean up utils.R

### PR DESCRIPTION
**Pull request by CrystalXuR**
Removed all unnecessary functions in the utils.R

### CODE REVIEW

File: survlearners/R/utils.R [hunk 1]
```diff
@@ -97,168 +90,3 @@ sanitize_input = function(x,w,y,D) {
 	            y=y,
 	            D=D))
 }
-
-#' @title data simulation
-#'
-#' @description Generates a dataset of size \eqn{n} that can be used to experiment with the learners and meta-learners
```
Comment (erikcs) [inline survlearners/R/utils.R:103]: When making changes to places in R code that touches these Roxygen entries:
Remember to run Build->Document and commit the updated .Rd files

Comment (erikcs) [reply survlearners/R/utils.R:103]: You can add that to your workflow notes?

Comment (CrystalXuR) [reply survlearners/R/utils.R:103]: yes, just figured it out, thanks

**Comment by erikcs** [comment]
If I write this in a comment or the description:

Closes #48 


the corresponding issue will be closed automatically when you merge the pr

**Review by erikcs — APPROVED** [review]

**Comment by CrystalXuR** [comment]
I see... good to know, thanks

**Comment by erikcs** [comment]
remember to delete the old unused branches

**Pull request merged**